src/terminal-config.h

```diff
@@ -40,6 +40,7 @@ typedef struct _StultoTerminalConfig {
     gboolean mouse_autohide;
     gboolean sync_clipboard;
     gboolean urgent_on_bell;
+    gboolean enable_headerbar;
```

Comment:
Not that we don't already have a few such settings, but the addition of this one calls out why it's a good idea to eventually implement either DConf/GSettings or be prepared to deal with a config hierarchy (à la global Stulto config and terminal profile configs).

This is not a setting we want to re-expose for the current app instance. Opening a new terminal tab should have no influence on the window state apart from adding a terminal, so this is ultimately not the correct place for this setting to live.